scripts/get-auto-track-script.ts

```diff
@@ -0,0 +1,56 @@
+/**
+ * Copyright IBM Corp. 2025, 2025
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+//cspell: disable
+export const getAutoTrack = (source: string) => `
+  <script>
+    window._ibmAnalytics = {
+      settings: {
+        name: '${source}',
+        isSpa: false,
+        tealiumProfileName: 'in-app-usage-growth',
+      },
+      onLoad: [['ibmStats.pageview', []]],
+    };
+
+    window.digitalData = {
+      page: {
+        pageInfo: {
+          ibm: {
+            siteId: ${`IBM_${source}`},
+          },
+          segment: {
+            enabled: true,
+            env: 'test',
+            key: 'aWg1ug3bDNcSTnde9iwvg1OcvVRzf0JC',
+            coremetrics: false,
+            commonProperties: {
```

Comment:
Just noting that @matthewgallo @jlongshore and @shawnkoid worked with CIO on how we set these. While we don’t have an official `UT30` or `productCode`, we set these so our events will be properly tracked and populated.

Reply:
^^ Just chatted more with Kevin from CIO, he is changing our plan type for our segment source so that we _only_ have to provide the `productCode` and `productTitle` properties.